### ISSUE TITLE
can't locate jstock.jar

### ISSUE THREAD

**Issue by mnhan32** [yccheok/jstock#69]
Hey, I follow [this instruction](http://jstock.org/help_install_upgrade.html)
It seems that I can't locate any jstock.jar in folders.

```
./jstock.sh 
Error: Unable to access jarfile jstock.jar
```
I am running archlinux with oracle java
```
archlinux-java status
Available Java environments:
  java-8-jdk (default)
```

**Comment by potholiday** [comment]
I dont know about archlinux but I am using ubuntu and I too faced the same problem. In ubuntu its because of how jstock creates **.desktop** file. It automatically assumes that we would install jstock in the root of home directory. I solved this problem by editing the **filename.desktop** file created by jstock. 

This is how I solved my problem in my system

first go to this location

`/home/.local/share/applications`

then find the jstock desktop file similar to this **org-yccheok-jstock-gui-jstock.desktop** . drag and drop in an editor and then edit the **path**  to the location where ever you installed your jstock
`path=/home/usr/fldr/jstock/`

this worked for me
